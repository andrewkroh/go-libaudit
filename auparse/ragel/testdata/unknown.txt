type=UNKNOWN[1333] msg=audit(1642008700.569:2424985): op=freq old=190270475413120 new=190270282200256