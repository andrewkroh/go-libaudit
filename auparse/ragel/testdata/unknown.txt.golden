type: ""
typeid: 1333
timestamp: "1642008700.569"
sequence: "2424985"
values:
  new: "190270282200256"
  old: "190270475413120"
  op: freq
message: ""
original: 'type=UNKNOWN[1333] msg=audit(1642008700.569:2424985): op=freq old=190270475413120 new=190270282200256'
